src/bundle/Resources/public/js/scripts/admin.input.text.js

```diff
@@ -0,0 +1,13 @@
+(function (global, doc, $, eZ) {
+    const textInputClearBtns = doc.querySelectorAll('.ibexa-input-text-wrapper__clear-btn');
+    const clearText = (event) => {
+        const inputWrapper = event.target.closest('.ibexa-input-text-wrapper');
+        const textInput = inputWrapper.querySelector('.ibexa-input--text');
+
+        textInput.value = '';
+        textInput.dispatchEvent(new Event('input'));
+        textInput.select();
```

Comment:
Is this select needed, you clear value after all?

Reply:
We want to select input after clicking the clear button. You can check out this behaviour e.g. on this page: https://www.diki.pl/slownik-angielskiego?q=asdf
after clicking the clear button "X" you can immediately start writing. The same effect we want to achieve here.